pylintrc_tweaks

```diff
@@ -1,4 +1,8 @@
-# pylintrc tweaks for use with edx_lint.
```

Comment:
Can we keep a comment in this file for context? I see that this change is related to another change to a file above - but it's not clear to me why they both exist. (Almost duplicate code changes)

Reply:
By other file do you mean `pylintrc`?  They are both generated by edx-lint, and the reason is that pylintrc would have all of the standard configuration information, and pylintrc_tweaks would have any repo-specific changes that are needed.

I'm not sure why the comment got deleted but I will put it back.